lib/index.d.ts

```diff
@@ -0,0 +1,286 @@
+// Type definitions for stardog.js 0.3.1
+// Project: https://github.com/stardog-union/stardog.js
+// Definitions by: Stephen Nowell <http://github.com/snowell>, Ty Soehngen <http://github.com/tysoeh>
```

Comment:
Can probably remove. Generally, I think you only add these when the defs don't live with the code and it does in this case.